static/src/javascripts/projects/common/modules/experiments/tests/contributions-story.js

```diff
@@ -0,0 +1,176 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
```

Comment:
This doesn't seemed to be used anywhere. No need to import modules that aren't being used